core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -73,18 +77,27 @@ var ExplorationEditorSettingsTab = function() {
 
   this.expectAvailableFirstStatesToBe = async function(names) {
     await waitFor.visibilityOf(
-      initialStateSelect,
-      'Initial state select takes too long to be visible.');
-    await waitFor.numberOfElementsToBe(
-      initialStateSelector, 'Options', 3, '<option>',);
-    var options = await initialStateSelect.$$('<option>')
-      .map(async function(elem) {
-        await waitFor.visibilityOf(
-          elem,
-          'option element taking too long to appear');
-        return await elem.getText();
-      });
+      initialStateSelect, 'Initial state select takes too long to be visible.');
+    let selectedStateName = await initialStateSelect.getText();
+
+    await action.click('State Dropdown element', initialStateSelect);
+
+    var options = await $$(
+      '.e2e-test-initial-state-select-element').map(async function(elem) {
+      await waitFor.visibilityOf(
+        elem,
+        'option element taking too long to appear');
+      return await elem.getText();
+    });
+
     expect(options.sort()).toEqual(names.sort());
+
+    // After opening dropdown and checking its options
+    // to close this dropdown we have written this piece
+    // of code.
+    await action.click(
+      'State name option', initialStateSelectOption(selectedStateName));
+    await action.click('Neutral element', neutralElement);
```

Comment:
Why is this added?

Reply:
after checking all the available options we need to close the dropdown, this code closes the dropdown. (it has been introduced because in angular material dropdown elements are only visible after click).

Reply:
after checking all the available options we need to close the dropdown, this code closes the dropdown. (it has been introduced because in angular material dropdown elements are only visible after click).